package.json

```diff
@@ -85,8 +85,8 @@
     "computeBenchmarkIndex": "./core/scripts/benchmark.js",
     "minify-latest-run": "./core/scripts/lantern/minify-trace.js ./latest-run/defaultPass.trace.json ./latest-run/defaultPass.trace.min.json && ./core/scripts/lantern/minify-devtoolslog.js ./latest-run/defaultPass.devtoolslog.json ./latest-run/defaultPass.devtoolslog.min.json",
     "save-latest-run": "./core/scripts/save-latest-run.sh",
-    "compile-proto": "protoc --python_out=./ ./proto/lighthouse-result.proto && mv ./proto/*_pb2.py ./proto/scripts || (echo \"❌ Install protobuf ≥ 3.7.1 to compile the proto file.\" && false)",
-    "build-proto-roundtrip": "mkdir -p .tmp && cross-env PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2 python proto/scripts/json_roundtrip_via_proto.py",
+    "compile-proto": "protoc --python_out=./ ./proto/lighthouse-result.proto && mv ./proto/*_pb2.py ./proto/scripts || (echo \"❌ Install protobuf = 3.20.x to compile the proto file.\" && false)",
+    "build-proto-roundtrip": "mkdir -p .tmp && cross-env PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=3 python3 proto/scripts/json_roundtrip_via_proto.py",
```

Comment:
I'm pretty sure `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION` should be `2` (it's the protobuf implementation version), but searching now since the tests passed, I don't think setting it actually does anything. Reading the value was removed in 3.19 (https://github.com/protocolbuffers/protobuf/pull/9090/) but even before that, the only value accepted was `2` (which was also the default value) since at least 2012, so I don't think it was ever needed.

#11087 added it to replace `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp`, which does actually do something but apparently didn't work on windows, but it doesn't appear to be necessary ([it appears `cpp` is the default for us](https://github.com/protocolbuffers/protobuf/blob/3e5fe0061f9b3e9263a50369e7ddab8f105d109f/python/google/protobuf/internal/api_implementation.py#L40-L92)).

I believe we can `yarn remove cross-env` at that point, as well.